Dockerfile

```
@@ -1,22 +1,18 @@
-# syntax=docker/dockerfile:experimental
-# Enables Docker buildkit caching feature, also requires setting DOCKER_BUILDKIT=1
+# syntax=docker/dockerfile:1.2
 FROM python:3.9-slim AS builder
 
-RUN --mount=type=cache,target=/root/.cache pip install poetry
+RUN --mount=type=cache,target=/root/.cache pip install poetry==1.1.10
 RUN poetry config virtualenvs.create false
 
-COPY poetry.lock .
-COPY pyproject.toml .
+COPY poetry.lock pyproject.toml ./
 
 RUN --mount=type=cache,target=/root/.cache poetry install --no-ansi
```

We should add the `--no-dev` option to make the image smaller.

```suggestion
RUN --mount=type=cache,target=/root/.cache poetry install --no-ansi --no-dev
```

---

I'm trying to get it to work, I'll update the Dockerfile as soon as I do, there seems to be a problem with the `certifi` package although I'm not sure why